Tests/Interactive/Common/GameDebug.cs

```diff
@@ -0,0 +1,52 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System.Collections.Generic;
+
+namespace MonoGame.InteractiveTests
+{
+    /// <summary>
+    /// Allows tests to output console messages including spammy messages that
+    /// may be throttled.
+    ///
+    /// On various platforms, this may be available via console output or via
+    /// a special console viewer (Console app on Mac; `adb logcat` on Android etc).
+    /// </summary>
+    public partial class GameDebug
+    {
+        /// <summary>Output a single console message.</summary>
+        public static void C(string message)
+        {
+            System.Console.WriteLine($"MGDBG: {message}");
+        }
+
+        /// <summary>Output an error message to the console.</summary>
+        public static void E(string message)
```

Comment:
Could we get more descriptive method names for these? `LogError` or `LogDebug`

Reply:
I am mimicking Log.e/i from Android where it's easy to write/ready in a concise manner.
In particular, I am using these as `GD.E` or `GD.I` and hoping the xmldoc helps clarify...